src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -71,12 +72,39 @@ class NetworkOPsImp final
     : public NetworkOPs
     , public beast::DeadlineTimer::Listener
 {
-public:
-    enum Fault
+    /**
+     * Transaction with input flags and results to be applied in batches.
+     */
+    class TransactionStatus
+    {
+    public:
+        Transaction::pointer transaction;
```

Comment:
When I see a struct, I think in terms of C, where the struct has no functions. It's a relic, and I think should be used in C++ only when it's still implemented as it would have been in C. Why use this thing at all still? To save from typing 8 characters?